presto-docs/src/main/sphinx/release/release-0.227.rst

```diff
@@ -0,0 +1,39 @@
+=============
+Release 0.227
+=============
+
+General Changes
+_______________
+* Enable spatial joins (broadcast and distributed) for ST_Equals, ST_Overlaps, ST_Crosses, and ST_Touches.
+* Fix a bug where index joins would fail with the error "driver should never block".  Now queries will wait until a configured timeout for the indexes to load.  The timeout can be configured with the configuration property ``index-loader-timeout`` and the session property ``index_loader_timeout``.
+* Move ``AggregationNode`` to SPI. Connectors can now push down aggregation to table scan.
+* Move ``ProjectNode`` to SPI. Connectors can now push down project to table scan.
+* Add ``task_partitioned_writer_count`` session property to allow to set number of concurrent writers for partitioned (bucketed) writes independently.
+* Removing syntax support for ``SET PATH``, ``CURRENT_PATH`` (Presto server do not use this information)
+* Fix NullPointerException in spatial join with geometry collections in the build side.
```

Comment:
```
NullPointerException
```

Reply:
```
on the build side
```

Reply:
> ```
> NullPointerException
> ```

@caithagoras 
Not sure what you meant here, but I changed the sentence to: 
```
* Fix a bug that could cause NullPointerException in spatial join with geometry collections on the build side.
```

Reply:
I meant that you need double back ticks around NullPointerException